main.py

```diff
@@ -38,14 +38,19 @@ def main(input_code, input_lang, output_lang):
         # TODO: send 400 client error
         return "Error must specify output language. For example, js for javascript and py for python"
    
-    # don't return error without sending to database
-    output_code = ""
+
     if (type(gast) == str) :
-        # return error if gast not built
-        output_code = "Error: did not compile"
-    else:
-        output_code = gtc.gast_to_code(gast, output_lang)
+        # return error if gast not built - dont store in database
```

Comment:
why not store this in the database? Wouldn't it be a useful thing to know if people aren't able to get code to compile?

Reply:
Good idea - going to store and categorize whether successful, didn't compile, or feature not supported